dist/aurelia-templating.d.ts

```diff
@@ -908,8 +908,6 @@ export declare class SlotCustomAttribute {
   constructor(element?: any);
   valueChanged(newValue?: any, oldValue?: any): any;
 }
-
```

Comment:
@jods4, i think You should revert all changes to generated code (last time i checked, generating code was only part of release process)